src/sentry/utils/snuba.py

```diff
@@ -2018,15 +2029,55 @@ def process_value(value: None | str | int | float | list[str] | list[int] | list
     return value
 
 
-# FLAGS/TAGS QUERY REWRITES.
-#
-# This is legacy code that I've overridden to rewrite tags queries. Flags are queried
-# identically to tags but they live on a different column. So we "OR" conditions where
-# a tag is present with a flag query.
-#
-# There is a more modern version of this code when the "organizations:issue-search-snuba"
-# flag is enabled. Everything below this comment can be removed when its permanently on.
-# No one else should be depending on these functions.
+# Flags and tags query re-writes. Flags are treated like tags but they're stored on a separate
+# column. We re-write queries targetting tags to also target flags. For now there is no explicit
+# search syntax for searching flags. Everything is done through the tags syntax.
+
+
+def error_stats_query_with_override(
```

Comment:
Nit
```suggestion
def error_stats_query_with_flags_override(
```
or 
```suggestion
def error_stats_query_with_flags(
```
Similar for `bulk_raw_query_with_override`, feel like we can make it clearer these are overrides are just for FFs.

Also curious, how soon do you expect the rewrites + search flag to be moved to the main codepath? For something like that do we have to do a gradual rollout?